packages/lit/src/test/node-imports.ts

```diff
@@ -45,3 +45,7 @@ import 'lit/html.js';
 import 'lit/experimental-hydrate-support.js';
 import 'lit/experimental-hydrate.js';
 import 'lit/static-html.js';
+
+import assert from 'node:assert/strict';
```

Comment:
Oo! This must be the new Node testing library.

Reply:
Just the [`assert`](https://nodejs.org/api/assert.html#assert) that's available globally in Node. The new [test runner](https://nodejs.org/api/test.html#test-runner) is still considered experimental but looks neat!